spec/features/scheduled_task_spec.rb

```diff
@@ -1,132 +1,86 @@
 require 'rails_helper'
+def enqueue_scheduled_tasks(stage)
+  # due_at = DateTime.now.in_time_zone + 120
+  # seconds_until_due = due_at - Time.now
+  # minutes_until_due = seconds_until_due / 60
+  id = 2
+  @name = "user"
+  due_at = DateTime.now.in_time_zone + 1.day
+  # puts DateTime.now.in_time_zone
+  # puts due_at
+  due_at1 = Time.zone.parse(due_at.to_s(:db))
+  curr_time = DateTime.now.in_time_zone.to_s(:db)
+  curr_time = Time.zone.parse(curr_time)
+  time_in_min = ((due_at1 - curr_time).to_i / 60) * 60
+  Delayed::Job.delete_all
+  expect(Delayed::Job.count).to eq(0)
+  Delayed::Job.enqueue(payload_object: ScheduledTask.new(id, stage, due_at), priority: 1, run_at: time_in_min)
+end
 
 describe 'Submission deadline reminder email' do
   it 'is able to send reminder email for submission deadline to signed-up users ' do
     # Delayed::Worker.delay_jobs = false
-    id = 2
-    @name = "user"
-
-    # due_at = DateTime.now.in_time_zone + 120
-    # seconds_until_due = due_at - Time.now
-    # minutes_until_due = seconds_until_due / 60
-    due_at = DateTime.now.in_time_zone + 1.day
-
-    # puts DateTime.now.in_time_zone
-    # puts due_at
-    due_at1 = Time.parse(due_at.to_s(:db))
-    curr_time = DateTime.now.in_time_zone.to_s(:db)
-    curr_time = Time.parse(curr_time)
-    time_in_min = ((due_at1 - curr_time).to_i / 60) * 60
-    Delayed::Job.delete_all
-    expect(Delayed::Job.count).to eq(0)
-
-    Delayed::Job.enqueue(payload_object: ScheduledTask.new(id, "submission", due_at), priority: 1, run_at: time_in_min)
-
+    # id = 2
```

Comment:
Please check the whole file `spec/features/delayed_mailer_spec.rb` and remove all of them.